packages/instrumentation-express/src/express.ts

```diff
@@ -227,7 +228,8 @@ export class ExpressInstrumentation extends InstrumentationBase<typeof express>
                 const routeAttributes = getRouteAttributes(routeState);
                 const route = routeAttributes[SemanticAttributes.HTTP_ROUTE] as string;
                 if (route) {
-                    req.__ot_middlewares = [route];
+                    const rpcMetadata = getRPCMetadata(context.active());
+                    rpcMetadata.route = route;
```

Comment:
Check for null. The user might not install HTTP instrumentation, or something might have messed up the context!